### ISSUE TITLE
Where can I find the implemented GMM attention for tacotron2?

### ISSUE THREAD

**Issue by xiaoyangnihao** [TensorSpeech/TensorFlowTTS#265]

**Comment by dathudeptrai** [comment]
I think there are some implementation of GMM based on pytorch/TF1 :)) you can refer it :v.

**Comment by xiaoyangnihao** [comment]
> I think there are some implementation of GMM based on pytorch/TF1 :)) you can refer it :v.

![image](https://user-images.githubusercontent.com/14233490/93582830-974d5780-f9d5-11ea-9088-c3024c7d0d80.png)

In my GMM attention，the mel_loss erro increases aperiodically, can you give some advices?

**Comment by dathudeptrai** [comment]
@xiaoyangnihao sorry to reply late, i think the problem is because learning rate, try to reduce it at the step the model's loss exploded.

**Comment by xiaoyangnihao** [comment]
thanks, i will try
